exercises/series/src/test/java/SeriesTest.java

```diff
@@ -17,145 +17,122 @@
-        List<Integer> actual = series.getDigits();
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
+    public void slicesOfOneFromTwo() {
+        Series series = new Series("12");
+        List<List<String>> expected = Arrays.asList(
+                Arrays.asList("1"), 
+                Arrays.asList("2")
+        );
+        List<List<String>> actual = series.slices(1);
         assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void keepsArbitraryDigitOrder() {
-        Series series = new Series("936923468");
-        List<Integer> expected = Arrays.asList(9, 3, 6, 9, 2, 3, 4, 6, 8);
-        List<Integer> actual = series.getDigits();
+    public void slicesOfTwo() {
+        Series series = new Series("35");
+        List<List<String>> expected = Arrays.asList(Arrays.asList("3", "5"));
+        List<List<String>> actual = series.slices(2);
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void canSliceByOne() {
-        Series series = new Series("01234");
-        List<List<Integer>> expected = Arrays.asList(
-		 Collections.singletonList(0),
-                Collections.singletonList(1),
-                Collections.singletonList(2),
-                Collections.singletonList(3),
-                Collections.singletonList(4)
+    public void slicesOfTwoOverlap() {
+        Series series = new Series("9142");
+        List<List<String>> expected = Arrays.asList(
+                Arrays.asList("9", "1"), 
+                Arrays.asList("1", "4"), 
+                Arrays.asList("4", "2")
         );
-        List<List<Integer>> actual = series.slices(1);
+        List<List<String>> actual = series.slices(2);
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void canSliceByTwo() {
-        Series series = new Series("98273463");
-        List<List<Integer>> expected = Arrays.asList(
-                Arrays.asList(9, 8),
-                Arrays.asList(8, 2),
-                Arrays.asList(2, 7),
-                Arrays.asList(7, 3),
-                Arrays.asList(3, 4),
-                Arrays.asList(4, 6),
-                Arrays.asList(6, 3)
-        );
-        List<List<Integer>> actual = series.slices(2);
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
-        assertEquals(expected, actual);
+    public void slicesIncludeDuplicates() {
+        Series series = new Series("777777");
+        List<List<String>> expected = Arrays.asList(
+                Arrays.asList("7", "7", "7"),
+                Arrays.asList("7", "7", "7"), 
+                Arrays.asList("7", "7", "7"), 
+                Arrays.asList("7", "7", "7")
+		);
```

Comment:
This line and the ones below for this test appear to be indented four spaces too much 🙂